app/src/androidTest/java/ch/sdp/vibester/activity/ProfileActivityTest.kt

```diff
@@ -232,6 +232,27 @@ class ProfileActivityTest {
         onView(withId(R.id.username)).check(matches(withText("Lalisa Bon")))
     }
 
+    @Test
+    fun checkChangePhotoCancel() {
+        val inputProfile = User( "Lalisa Bon","bit.ly/3IUnyAF", "lisa@test.com",  12, 8, "VvPB47tQCLdjz3YebilS6h5EXdJ3")
+        val ctx = ApplicationProvider.getApplicationContext() as Context
+        val intent = Intent(ctx, ProfileActivity::class.java)
+
+        createMockDataGetter(inputProfile)
+        createMockAuthenticator()
+        createMockImageGetter()
+
+        val scn: ActivityScenario<ProfileActivity> = ActivityScenario.launch(intent)
+
+        //This thread sleep is added for the mock image to load, might be a better way to test it but for now I'll leave it like that
+        Thread.sleep(1000)
```

Comment:
Reminder to take a look at it just in case. If you're gonna do it in the future, better put a TODO instead